### ISSUE TITLE
Just a thought...

### ISSUE THREAD

**Issue by r0t0tiller** [leonli96/python-hacklib#10]
Once we build this up more and we have a solid foundation, it would be great to make a wiki for this. If you want people to use it, we gotta make some form of documentation. README looks great so far, but it would be nice to create a document that explains each function.

**Comment by lazorfuzz** [comment]
Yeah, I was hoping to add a wiki once it got too ridiculous to put everything in the README haha.

**Comment by r0t0tiller** [comment]
Haha alright sounds like a plan!

**Comment by TTcoinDeveloper** [comment]
Interesting 🤐

**Comment by TTcoinDeveloper** [comment]
@lazorfuzz  @r0t0tiller I could help you with the wiki and any help u need. I want to send an Idea of how start life on Mars to SpaceX.
I think it will help them greatly.
If I didn't believe in that project.
I wouldn't ask for help.
I certainly don't have to to send anything.
They have a team 
But if it makes a little bit of a difference
It still worth it that I know I tried.
